### PR TITLE
Add new helper functions

### PR DESCRIPTION
**Pull request by priyalalK**

### CODE REVIEW

File: specs/expectToBe.spec.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/*global describe, expect, require, beforeEach, browser, xit, it, element, by, beforeAll*/
+
+(function () {
+    "use strict";
+
+    var ExpectHelperPage = require('../helper/expectHelper.js'),
+        expectHelperPage = new ExpectHelperPage(),
+
```
Comment (kasunkv) [inline specs/expectToBe.spec.js]: I don;t see the issue here.. Am i missing something.. A little more details pls..

**Comment by charithsoori** [comment]
PR#26 included  code checking . Please accept it

**Comment by rehrumesh** [comment]
@priyalalK please rebase and update the PR as current work is outdated.

**Comment by kasunkv** [comment]
These helpers will be converted in to TypeScript and the JS file will be removed. Any code quality issues will be fixed there..

**Pull request merged**